app/ui/omni-box.js

```diff
@@ -1,5 +1,7 @@
 /* global HTMLElement, CustomEvent, customElements */
 
+const { looksLikeLegacySSB, convertLegacySSB: makeSSB } = require('ssb-fetch')
```

Comment:
I think the error you has in the other GH issue was related to the `debug` module not working well inside Electron's renderer process. Would it be possible to split out these two functions into their own module and import that, or just define some of these functions within the omnibox itself?

Reply:
Disregard this, the requires are fine as they are.